source/mongodb/data-access.txt

```diff
@@ -0,0 +1,124 @@
+.. _client-data-access:
+
+============================
+Query Atlas from Client Apps
+============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can query data in MongoDB Atlas from client applications using standard
+MongoDB query language syntax with **MongoDB Data Access** in the Realm SDKs.
+
+The MongoDB Data Access secures data with role-based App Services rules, schemas,
+and filters. These features let construct complex client-side queries
+while your data remains secure on the server.
```

Comment:
Can we indicate somewhere here that you use MQL on linked data sources? (i.e. you can't just use it on any cluster, you have to make the cluster available to your App)

Reply:
added to 1st sentence of page